### PR TITLE
add screenshots of dashboard operations for docs of some plugins

### PR DESCRIPTION
**Pull request by fc13240**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/iresty/apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

add screenshots of dashboard operations for docs of some plugins

### CODE REVIEW

File: doc/plugins/ip-restriction-cn.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+$ curl http://127.0.0.1:9080/index.html
+HTTP/1.1 200 OK
+...
+```
+
+访问 `127.0.0.2`:
+
+```shell
+$ curl http://127.0.0.2:9080/index.html -i
+HTTP/1.1 403 Unauthorized
```
Comment (juzhiyuan) [inline doc/plugins/ip-restriction-cn.md]: 401 status code is more suitable

File: doc/plugins/ip-restriction-cn.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+...
+{"message":"Your IP address is not allowed"}
```
Comment (juzhiyuan) [inline doc/plugins/ip-restriction-cn.md]: consider adding a space like `{"message": "Your XXX"}` ?

File: doc/plugins/ip-restriction-cn.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+## 禁用插件
+
+当你想去掉 ip-restriction 插件的时候，很简单，在插件的配置中把对应的 json 配置删除即可，无须重启服务，即刻生效：
```
Comment (juzhiyuan) [inline doc/plugins/ip-restriction-cn.md]: consider using Markdown Highlight format for `ip-restriction` :)

File: doc/plugins/ip-restriction-cn.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "39.97.63.215:80": 1
+        }
+    }
+}'
+```
+
+现在就已经移除了 ip-restriction 插件了。其他插件的开启和移除也是同样的方法。
```
Comment (juzhiyuan) [inline doc/plugins/ip-restriction-cn.md]: 1. change `他` to `它`
2. make the statement more fluent~~

File: doc/plugins/jwt-auth-cn.md [hunk 5]
```diff
@@ -39,6 +39,11 @@ curl http://127.0.0.1:9080/apisix/admin/consumers -X PUT -d '
     }
 }'
 ```
+你可以使用浏览器打开 dashboard：`http://127.0.0.1:9080/apisix/dashboard/`，通过 web 界面来完成上面的操作，先增加一个 consumer：
```
Comment (juzhiyuan) [inline doc/plugins/jwt-auth-cn.md]: e.g 你可以访问 Dashboard `http://127.0.0.1:9080/apisix/dashboard/` 并通过 Web 控制台来增加一个 `Consumer`：

File: doc/plugins/ip-restriction-cn.md (hunk 1 above)
Comment (membphis) [reply doc/plugins/ip-restriction-cn.md]: good cache, I'll fix it later.

**Comment by membphis** [comment]
@fc13240 thx for your great job ^_^

**Comment by moonming** [comment]
@fc13240 please take a look above comments. thx

**Comment by membphis** [comment]
merged first, I'll fix some small problem later.

File: doc/plugins/ip-restriction-cn.md (hunk 2 above)
Comment (membphis) [reply doc/plugins/ip-restriction-cn.md]: it encoded by `cjson` library by machine.

**Pull request merged**